CHANGES.md

```diff
@@ -85,12 +85,14 @@ Release Notes.
 * Support large service/instance/networkAddressAlias list query by using ElasticSearch scrolling API, add `metadataQueryBatchSize` to configure scrolling page size.
 * Change default value of `metadataQueryMaxSize` from `5000` to `10000`
 * Replace deprecated Armeria API `BasicToken.of` with `AuthToken.ofBasic`.
+* Implement v9 UI template management protocol.
```

Comment:
A reminder, we should overhaul https://github.com/apache/skywalking/tree/master/dist-material/release-docs/licenses/ui-licenses accordingly as the UI project is completely replaced and must have some dependencies' changes

Reply:
Yes, we will.

Reply:
FYI @Fine0830

Reply:
Created https://github.com/apache/skywalking/issues/8638 to track this task.